tests/functional/cylc-reinstall/00-simple.t

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env bash
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#------------------------------------------------------------------------------
+# Test workflow re-installation
+. "$(dirname "$0")/test_header"
+set_test_number 44
+
+# Test basic cylc reinstall, named run given 
+TEST_NAME="${TEST_NAME_BASE}-basic-named-run"
+make_rnd_suite
+pushd "${RND_SUITE_SOURCE}" || exit 1
+run_ok "${TEST_NAME}" cylc install
+contains_ok "${TEST_NAME}.stdout" <<__OUT__
+INSTALLED $RND_SUITE_NAME from ${RND_SUITE_SOURCE} -> ${RND_SUITE_RUNDIR}/run1
+__OUT__
+run_ok "basic-reinstall" cylc reinstall "${RND_SUITE_NAME}/run1"
+REINSTALL_LOG="$(find "${RND_SUITE_RUNDIR}/run1/log/install" -type f -name '*reinstall.log')"
+grep_ok "REINSTALLED ${RND_SUITE_NAME}/run1 from ${RND_SUITE_SOURCE} -> ${RND_SUITE_RUNDIR}/run1" "${REINSTALL_LOG}" 
+
+popd || exit 1
+purge_rnd_suite
+
+# Test basic cylc reinstall, named run (including flow name) given 
```

Comment:
petty:
```suggestion
# Test basic cylc reinstall, named run (including ``flow.cylc`` name) given.
```

Reply:
I think it is a flow name (i.e. reg) rather than a `flow.cylc` path?

Reply:
Line 43?